core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -311,6 +311,32 @@ export class RteHelperModalComponent {
     return url !== text;
   }
 
+  validateVideoStartEnd(): boolean {
+    const tmpCustomizationArgs = this.tmpCustomizationArgs as Extract<
+      CustomizationArgsNameAndValueArray[number],
+      {
+        name: 'video_id' | 'start' | 'end' | 'autoplay';
+      }
+    >[];
+    let start: number = 0;
+    let end: number = 0;
+    for (const arg of tmpCustomizationArgs) {
+      if (arg.name === 'start') {
+        if (typeof arg.value === 'number') {
```

Comment:
If they are not a number shouldn't we also return error?

Reply:
Did some testing and input can only be submitted in number so removed this check since it will never be used